encoding/wkb/sql.go

```diff
@@ -67,6 +77,32 @@ type GeometryCollection struct {
 	opts []wkbcommon.WKBOption
 }
 
+// Scan scans from a []byte.
+func (g *Geom) Scan(src interface{}) error {
+	b, ok := src.([]byte)
+	if !ok {
+		return ErrExpectedByteSlice{Value: src}
+	}
+	// NOTE(tb) other Scanners do not check the len of b, is it really useful ?
+	if len(b) == 0 {
+		return nil
+	}
+	var err error
+	g.T, err = Unmarshal(b, g.opts...)
+	return err
+}
+
+// Value returns the WKB encoding of g.
+func (g *Geom) Value() (driver.Value, error) {
+	return value(g.T)
+}
+
+// NOTE(tb) you asked for this method but since accessing the geom.T is trivial
+// as writing g.T, I wonder if it worth it.
```

Comment:
I agree it's trivial, but I think it will look prettier in code as it better indicates what is returned compared to just `.T` alone.

Reply:
Fine, I replaced the note with a godoc

92b5cec